### PR TITLE
Add Supervisor Barge Coach Features

### PR DESCRIPTION
**Pull request by aestellwag**
Added Supervisor Barge Coach + Extras (as toggled features)

### CODE REVIEW

**Comment by jhunter-twilio** [comment]
A rebase from main, some minor refactoring for the serverless functions and the other commends and then lastly, could you add a readme in, complete with a video recording visualizing the flex user experience, same format as the other feature library readmes (except activity filter which needs updated)

**Comment by aestellwag** [comment]
Today's push includes rebase, twilio wrapper, small bug fix, and added the Readme

File: plugin-flex-ts-template/src/flex-hooks/components/MainHeader.tsx [hunk 1]
```diff
@@ -1,6 +1,4 @@
 import * as Flex from '@twilio/flex-ui';
-import { replaceUserControls } from '../../feature-library/activity-skill-filter/flex-hooks/components/MainHeader'
```
Comment (jhunter-twilio) [inline plugin-flex-ts-template/src/flex-hooks/components/MainHeader.tsx:2]: Can you revert these changes?

File: plugin-flex-ts-template/src/flex-hooks/components/WorkerCanvas.tsx [hunk 2]
```diff
@@ -1,6 +1,4 @@
 import * as Flex from '@twilio/flex-ui';
-import { replaceWorkerCanvasProfile } from '../../feature-library/activity-skill-filter/flex-hooks/components/WorkerCanvas';
```
Comment (jhunter-twilio) [inline plugin-flex-ts-template/src/flex-hooks/components/WorkerCanvas.tsx:2]: these too

File: plugin-flex-ts-template/src/flex-hooks/components/NoTasksCanvas.tsx [hunk 3]
```diff
@@ -1,6 +1,4 @@
 import * as Flex from '@twilio/flex-ui';
-import { replaceAvailability } from '../../feature-library/activity-skill-filter/flex-hooks/components/NoTasksCanvas';
```
Comment (jhunter-twilio) [inline plugin-flex-ts-template/src/flex-hooks/components/NoTasksCanvas.tsx:2]: these too

File: plugin-flex-ts-template/src/flex-hooks/components/TaskCanvasHeader.tsx [hunk 4]
```diff
@@ -1,7 +1,6 @@
 import * as Flex from '@twilio/flex-ui';
 import { addTransferButtonToChatTaskView } from '../../feature-library/chat-transfer/flex-hooks/components/TaskCanvasHeader'
```
Comment (jhunter-twilio) [inline plugin-flex-ts-template/src/flex-hooks/components/TaskCanvasHeader.tsx:2]: these too

File: plugin-flex-ts-template/src/flex-hooks/notifications/index.ts [hunk 5]
```diff
@@ -1,10 +1,8 @@
 import * as Flex from '@twilio/flex-ui';
-import ActivitySkillFilter from '../../feature-library/activity-skill-filter/flex-hooks/notifications/ActivitySkillFilter'
```
Comment (jhunter-twilio) [inline plugin-flex-ts-template/src/flex-hooks/notifications/index.ts:2]: revert these

File: plugin-flex-ts-template/src/flex-hooks/strings/index.ts [hunk 6]
```diff
@@ -1,12 +1,10 @@
 import * as Flex from '@twilio/flex-ui';
-import ActivitySkillFilter from '../../feature-library/activity-skill-filter/flex-hooks/strings/ActivitySkillFilter'
```
Comment (jhunter-twilio) [inline plugin-flex-ts-template/src/flex-hooks/strings/index.ts:2]: revert these

File: serverless-functions/.env.dev [hunk 7]
```diff
@@ -1,9 +1,9 @@
 # ACCOUNT SPECIFIC SETTINGS
 
 # GENERAL 
-TWILIO_FLEX_WORKSPACE_SID=WSdda966af760d30cca8dbd064987673cf
```
Comment (jhunter-twilio) [inline serverless-functions/.env.dev:4]: revert changes to this file

**Review by jhunter-twilio — CHANGES_REQUESTED** [review]

File: serverless-functions/.env.dev (hunk 7 above)
Comment (aestellwag) [reply serverless-functions/.env.dev:4]: I intially removed the SIDs but believe you will be keeping those in as examples

File: flex-config/.env.dev [hunk 8]
```diff
@@ -0,0 +1,4 @@
+# ACCOUNT SPECIFIC SETTINGS
```
[the remark this answers is not in the document]
Comment (aestellwag) [reply flex-config/.env.dev:1]: Another odd file add/remove - this will be gone

**Review by jhunter-twilio — APPROVED** [review]
